kruize-clowdapp.yaml

```diff
@@ -79,7 +79,9 @@ objects:
             value: ${SSL_CERT_DIR}
     jobs:
       - name: delete-kruize-partitions
-        schedule: "0 0 16 * *" # Run on 16th of every month at midnight
+         # Apr 1st 2024 Editing to run the cronJob daily to delete partitions
+        schedule: "0 0 0 * *" # Run every day at midnight
```

Comment:
Hello @rbadagandi1, similar to this [change](https://github.com/RedHatInsights/ros-ocp-backend/pull/182/files#diff-0f518b5985b4ea86412d39c9757171411345db257cb061b84bb314a7a2b3d5e9R201), could you make this value configurable so that one can change it as per environment?

Reply:
Please validate the changes @kgaikwad